pkgs/servers/nominatim/default.nix

```diff
@@ -0,0 +1,66 @@
+{
+  stdenv, lib, fetchgit, fetchurl,
+  clang-tools, cmake, bzip2, zlib, expat, boost, git, pandoc, gzip,
+  postgresql_12,
+  python3, python3Packages, php
+}:
+
+let
+  countryGrid = fetchurl {
+    # Nominatim docs mention this URL but it's not a very good one for pinning
```

Comment:
there is web.archive.org for such cases https://web.archive.org/web/20220323041006/https://nominatim.org/data/country_grid.sql.gz

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/165183/commits/bc4bf3e4b6d33a1997f44b6a4f495717256eec6d thanks